training/src/anemoi/training/config/model/graphtransformer.yaml

```diff
@@ -60,13 +16,13 @@ processor:
   num_heads: 16 # GraphTransformer or Transformer only
   qk_norm: False
   cpu_offload: ${model.cpu_offload}
-
+  # LayerKernels are optional. If not specified, the default are pytorch.nn.LayerNorm and torch.nn.Linear.
+  # TODO: Add to documentation: The query and key norms need to be autocast layer norms if flash attention is used.
 
 encoder:
   _target_: anemoi.models.layers.mapper.GraphTransformerForwardMapper
   trainable_size: ${model.trainable_parameters.data2hidden}
   sub_graph_edge_attributes: ${model.attributes.edges}
-  activation: ${model.activation}
   num_chunks: 1
```

Comment:
Why in this case we don't have the layer kernels included? do we rely on the defaults?

Reply:
I was trying to reduce config bloat and rather add to the documentation instead. But I can change it if you insist.

Reply:
I disagree with this decision, but I will make the change.